projects/Exceptions/ElementsList/src/com/valentin/OutOfRange.java

```diff
@@ -0,0 +1,7 @@
+package com.valentin;
+
+public class OutOfRange extends IndexOutOfBoundsException {
```

Comment:
It's not a good idea to extend the IndexOutOfBoundsException. You can just extend RuntimeException.